test/hotspot/jtreg/compiler/sharedstubs/SharedStubToInterpTest.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+/**
+ * @test SharedStubToInterpTest
+ * @summary Checks that stubs to the interpreter can be shared for static or final method.
+ * @bug 8280481
+ * @library /test/lib
+ *
+ * @requires os.arch=="amd64" | os.arch=="x86_64" | os.arch=="i386" | os.arch=="x86" | os.arch=="aarch64"
+ *
+ * @run driver compiler.sharedstubs.SharedStubToInterpTest c2 StaticMethodTest
```

Comment:
Is not it better to run all the VM instances from a single test?
```java
List<String> options = java.util.Arrays.asList("-XX:-TieredCompilation", "-XX:TieredStopAtLevel=1");
List<String> tests = java.util.Arrays.asList("StaticMethodTest", "FinalClassTest", "FinalMethodTest");
for (String option: options) {
    for (String test: tests) {
        runVM(option, test);
    }
} 
```

Reply:
Done